test.py

```diff
@@ -16,6 +16,25 @@ def test_peers(self):
         self.assertTrue(type(result['peers']) == list)
         self.assertTrue(len(result['peers']) >= 1)
         self.assertEqual(len(result['peers']), result['count'])
+    
+    def test_search(self):
+        result= self.loklak.search('doctor who')
+        self.assertTrue('statuses' in result)
+        self.assertTrue(type(result['statuses']) == list)
+        self.assertTrue(len(result['statuses']) >= 1)
+        self.assertEqual(len(result['statuses']), int(result['search_metadata']['count']))
+
```

Comment:
Also test for `search` in case search failure, a failure assertion has to also be made for failed searches or searches which aren't made correctly.